Core/Core/Use Cases/UploadMedia.swift

```diff
@@ -54,14 +54,17 @@ public class UploadMedia: NSObject, URLSessionDelegate, URLSessionDataDelegate {
 
     func upload() {
         task = env.api.makeRequest(GetMediaServiceRequest()) { (data, _, error) in
-            guard
-                error == nil,
-                let domain = data?.domain.replacingOccurrences(of: "https://", with: ""),
-                let url = URL(string: "https://\(domain)")
-            else {
+            var url: URL?
+            if let domain = data?.domain,
+                domain.starts(with: "http://localhost:") {
```

Comment:
Do you want the colon on the end `localhost:` ?  Not sure if this was a typo or you have port too in url

Reply:
Yeah, there's a port. Just being paranoid about something like "localhost.notreally.com"